### ISSUE TITLE
Google Apps

### ISSUE THREAD

**Issue by farukaydin** [automatisch/automatisch#932]
### Discussed in https://github.com/automatisch/automatisch/discussions/754

<div type='discussions-op-text'>

<sup>Originally posted by **DoubyDoubs** November 29, 2022</sup>
It would be nice if we could add an integration with Google Apps. One of the main one could be Google Drive and Google Form. Automatize when someone if filling your form is quite handy!</div>

Existing upvote count from GitHub discussions: 18

**Comment by simonmueck** [comment]
Yeah I'd love to see that too! I was trying out some stuff between Notion and Google Drive. I wanted to rename and move a folder based on a status of a Database item in Notion. When setting up a trigger it would be great to change the folder name or folder location of a google drive folder :)
